parts/k8s/addons/kubernetes-dashboard.yaml

```diff
@@ -0,0 +1,319 @@
+
```

Comment:
Is there any difference in the dashboard specs between 1.14 and 1.19? Wondering if we can consolidate these into one file...

Reply:
This manifest works with versions 1.14+ in my testing (at least, the Dashboard is accessible after setting up a role and getting a token). This should be the only manifest we're using for Dashboard for all supported versions.

Reply:
Here is the compatibility matrix for Kubernetes Dashboard [v2.0.0](https://github.com/kubernetes/dashboard/releases/tag/v2.0.0):

Kubernetes version | 1.15 | 1.16 | 1.17 | 1.18
-- | -- | -- | -- | --
Compatibility | ? | ? | ? | ✓

✓ Fully supported version range.
? Due to breaking changes between Kubernetes API versions, some features might not work correctly in the Dashboard.

Reply:
oh sorry jeez

red means deleted

nothing to see here